### PR TITLE
React 723 UI enhancement

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
Project Assignments - Implement new date range picker
[#723](https://github.com/nerubia/kh360-react/issues/723)

### CODE REVIEW

File: src/features/admin/project-assigments/project-assignment-form/project-assignment-form.tsx [hunk 1]
```diff
@@ -446,7 +446,7 @@ export const ProjectAssignmentForm = () => {
 
   return (
     <div className='flex flex-col gap-10'>
-      <div className='flex flex-col xl:w-1/2 gap-4'>
+      <div className='flex flex-col xl:w-3/4 gap-4'>
```
Comment (gborgonia-ir) [inline src/features/admin/project-assigments/project-assignment-form/project-assignment-form.tsx:449]: can we retain 1/2?

Comment (chadiegilnerubia) [reply src/features/admin/project-assigments/project-assignment-form/project-assignment-form.tsx]: done :3

File: src/components/ui/input/input.tsx [hunk 2]
```diff
@@ -44,9 +46,9 @@ export const Input = ({
         </label>
       )}
       <input
-        className={`${error != null ? "border-red-500" : ""} border px-4 py-1.5 rounded-md ${
-          isMobile ? "text-sm" : "text-base"
-        }`}
+        className={`w-${width}${
```
Comment (emmaacha) [inline src/components/ui/input/input.tsx:49]: remove `w-` here and add space between `${width}` and `${`

File: src/features/admin/project-assigments/project-assignment-form/project-assignment-form.tsx [hunk 3]
```diff
@@ -550,6 +550,7 @@ export const ProjectAssignmentForm = () => {
                 onChange={handleInputChange}
                 error={validationErrors.allocation_rate}
                 max={100}
+                width='9/10'
```
Comment (emmaacha) [inline src/features/admin/project-assigments/project-assignment-form/project-assignment-form.tsx:553]: Make this width='w-9/10'

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/components/ui/input/input.tsx (hunk 2 above)
Comment (chadiegilnerubia) [reply src/components/ui/input/input.tsx]: done :3

**Review by emmaacha — APPROVED** [review]

**Review by gborgonia-ir — APPROVED** [review]

**Pull request merged**